backend/fleet_management/management/commands/populate_database.py

```diff
@@ -106,6 +107,9 @@ def info(message):
                 driver=random.choice(drivers),
                 car=random.choice(cars),
             )
+        info("Creating 50 random refuels")
+        for _ in tqdm(range(50)):
+            RefuelFactory.create()
```

Comment:
```suggestion
            RefuelFactory.create(driver=random.choice(drivers), car=random.choice(cars))
```
without that `driver=` and `car=` the RefuelFactory will create new users and cars instead of using the already generated ones.

Reply:
bump, that is still an issue to correct

Reply:
Done